src/omnibase_core/models/contracts/model_contract_compute.py

```
@@ -64,31 +64,77 @@ class ModelContractCompute(ModelContractBase):
+                    raise ModelOnexError(
+                        message=f"Invalid string value for node_type: {v}",
+                        error_code=EnumCoreErrorCode.VALIDATION_ERROR,
+                        details=ModelErrorContext.with_context(
+                            {
+                                "error_type": ModelSchemaValue.from_value("valueerror"),
+                                "validation_context": ModelSchemaValue.from_value(
+                                    "model_validation",
+                                ),
+                            },
+                        ),
+                    )
```

Ruff (0.14.7)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)